src/Components/CollectionsHubRails/ArtistSeriesRail/ArtistSeriesEntity.tsx

```diff
@@ -24,9 +25,24 @@ export const ArtistSeriesEntity: React.FC<ArtistSeriesEntityProps> = ({
   const bgImages = hits.map(hit => hit.image.url)
   const imageSize =
     bgImages.length === 1 ? 265 : bgImages.length === 2 ? 131 : 85
+  const { trackEvent } = useTracking()
+  const onClickLink = event => {
+    const link = event.target
+    const href = link.parentNode.parentNode.parentNode.getAttribute("href")
```

Comment:
Can this `href` be added to the graphql request below and then plucked from props?

Reply:
href is not available from the graphql request. 

![Screen Shot 2019-08-20 at 11 03 06](https://user-images.githubusercontent.com/45926410/63359130-35d23980-c33a-11e9-8a04-32ea3f35f39d.png)